ir/src/main/scala/org/scalajs/core/ir/Infos.scala

```diff
@@ -233,14 +271,21 @@ object Infos {
     }
 
     def result(): MethodInfo = {
+      def toMapOfLists(
+          m: mutable.Map[String, mutable.Set[String]]): Map[String, List[String]] = {
+        m.mapValues(_.toList).toMap
```

Comment:
That's why I now call `.mapValues` before calling `.toMap`. The latter will force, so the laziness is not an issue.